pkgs/development/haskell-modules/hackage-packages.nix

```diff
@@ -105979,6 +105979,26 @@ self: {
        hydraPlatforms = stdenv.lib.platforms.none;
      }) {};
 
+  "happy_1_19_9" = callPackage
```

Comment:
This file shouldn't be updated in PRs.  It is updated automatically:

https://github.com/NixOS/nixpkgs/pull/67438#issuecomment-525522019